tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -27,6 +29,7 @@ import (
     "../wski18n"
 
     "github.com/fatih/color"
+    "github.com/ghodss/yaml"
```

Comment:
It looks like this one: https://github.com/go-yaml/yaml/tree/v2
This is License Apache 2 https://github.com/go-yaml/yaml/blob/v2/LICENSE